backend/origuide/solver.py

```diff
@@ -64,6 +66,7 @@ def solve(self, output):
                 if plot_idx >= CONFIG['DEBUG_PLOT_FROM'] and plot_idx % CONFIG['DEBUG_PLOT_EVERY'] == 0:
                     plot.plot3d(self.vertices, self.edges, self.faces, cur_forces)
                 plot_idx += 1
+            print(plot_idx)
```

Comment:
Debug statement left dangling.

Reply:
It's not for debug only. I left it to show some progress as the solver is soooo slow for now.